src/app/chat/views/messages.nim

```diff
@@ -221,7 +234,7 @@ QtObject:
         let isEdit = msg.editedAt != "0" or msg.contentType == ContentType.Edit
         if not channel.muted and not isEdit and not isGroupSelf and not isMyInvite:
           let isAddedContact = channel.chatType.isOneToOne and self.isAddedContact(channel.id)
-          self.messageNotificationPushed(msg.id, channel.communityId, msg.chatId, escape_html(msg.text), msg.contentType.int, channel.chatType.int, msg.timestamp, msg.identicon, msg.userName, msg.hasMention, isAddedContact, channel.name)
```

Comment:
nit: no more escape_html

Reply:
escape_html not defined here and is used elsewhere. hence, didn't touch it